jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/RequestTest.java

```diff
@@ -372,6 +377,38 @@ public boolean handle(org.eclipse.jetty.server.Request request, Response respons
         assertThat(response.getStatus(), is(HttpStatus.OK_200));
     }
 
+    public static Stream<Arguments> localeTests()
+    {
+        return Stream.of(
+            Arguments.of(null, List.of(Locale.getDefault().toLanguageTag()).toString()),
+            Arguments.of("zz", List.of(Locale.US.toLanguageTag()).toString()),
+            Arguments.of("en", "[en]"),
+            Arguments.of("en-gb", List.of(Locale.UK.toLanguageTag()).toString()),
+            Arguments.of("en-us", List.of(Locale.US.toLanguageTag()).toString()),
+            Arguments.of("EN-US", List.of(Locale.US.toLanguageTag()).toString()),
+            Arguments.of("en-us,en-gb", List.of(Locale.US.toLanguageTag(), Locale.UK.toLanguageTag()).toString()),
+            Arguments.of("en-us;q=0.5,fr;q=0.0,en-gb;q=1.0", List.of(Locale.UK.toLanguageTag(), Locale.US.toLanguageTag()).toString())
```

Comment:
This is missing the check for the `fr` (French) tag.

Reply:
because it is "q=0.0" and thus excluded from the list!

Reply:
> because it is "q=0.0" and thus excluded from the list!

Ah, good point.